source/wp-content/themes/wporg-developer/header.php

```diff
@@ -24,33 +33,27 @@
 		<p class="site-description"><?php _e( 'The freedom to build.', 'wporg' ); ?></p>
 		<?php endif; ?>
 
-		<nav id="site-navigation" class="main-navigation" role="navigation">
-			<button class="menu-toggle dashicons dashicons-arrow-down-alt2" aria-controls="primary-menu" aria-expanded="false" aria-label="<?php esc_attr_e( 'Primary Menu', 'wporg' ); ?>"></button>
-			<?php
-			$active_menu = is_post_type_archive( 'command' ) || is_singular( 'command' ) ? 'devhub-cli-menu' : 'devhub-menu';
-			wp_nav_menu( array(
-				'theme_location'  => $active_menu,
-				'container_class' => 'menu-container',
-				'container_id'    => 'primary-menu',
-			) ); ?>
-		</nav>
+		<div>
+			<?php if ( $show_search ) : ?>
+				<?php get_search_form(); ?>
+			<?php endif; ?>
+
+			<nav id="site-navigation" class="main-navigation" role="navigation">
+				<button class="menu-toggle dashicons dashicons-arrow-down-alt2" aria-controls="primary-menu" aria-expanded="false" aria-label="<?php esc_attr_e( 'Primary Menu', 'wporg' ); ?>"></button>
+				<?php
+				$active_menu = is_post_type_archive( 'command' ) || is_singular( 'command' ) ? 'devhub-cli-menu' : 'devhub-menu';
+				wp_nav_menu( array(
+					'theme_location'  => $active_menu,
+					'container_class' => 'menu-container',
+					'container_id'    => 'primary-menu',
+				) ); ?>
+			</nav>
```

Comment:
If I add items to this menu, they show up kind of unexpectedly:

<img width="995" alt="Screen Shot 2022-06-09 at 3 24 04 PM" src="https://user-images.githubusercontent.com/541093/172928162-44404a9c-0602-4dc7-a2a8-e81809a8579a.png">

It looks like this menu is empty on production, so perhaps it's fine, but it might be good to at least make this presentable (or remove the menu). That can happen in a followup PR.